### PR TITLE
[FIX] sale_margin_percentage: Migrated attribute in element of list view

### PR DESCRIPTION
**Pull request by deivislaya**
vx#8738

**TODO**

- [x] Changed attribute in sale.order.line
- [x] Adding help to fields

### CODE REVIEW

**Comment by deivislaya** [comment]
Hi @JulioSerna

This PR is ready for review

**Comment by deivislaya** [comment]
Hi @suniagajose 

This PR is ready for review

File: sale_margin_percentage/models/sale_order.py [hunk 1]
```diff
@@ -33,17 +32,19 @@ class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
     @api.model
-    def _default_margin_threshold(self):
+    def _get_default_margin_threshold(self):
```
Comment (suniagajose) [inline sale_margin_percentage/models/sale_order.py]: @deivislaya 

please, leave this method as `_default_margin_threshold`..

by [convention](https://www.odoo.com/documentation/9.0/reference/guidelines.html#symbols-and-conventions) is the right name

regards,

Comment (deivislaya) [reply sale_margin_percentage/models/sale_order.py]: Thanks for the info, change applied

Comment (suniagajose) [reply sale_margin_percentage/models/sale_order.py]: +1

**Review by suniagajose — APPROVED** [review]

**Pull request merged**